optuna/distributions.py

```diff
@@ -231,17 +231,22 @@ class IntUniformDistribution(BaseDistribution):
             Lower endpoint of the range of the distribution. ``low`` is included in the range.
         high:
             Upper endpoint of the range of the distribution. ``high`` is included in the range.
+        q:
+            A step of discretization.
```

Comment:
The term "discretization" feels a bit odd to me because int values ​​are already discrete values even if this option isn't specified.

Reply:
Besides, I prefer "step" to "q" because, IMO, the former is a more intuitive name. But, I want to hear opinions from other developers and contributors.

Reply:
Your option looks reasonable. 

Talking of this implement, I use `q` as a step parameter because discrete_uniform's step argument is `q`. it's just a unification between `discrete_uniform` and `int`. 

Looking at other libraries, however, `step` is seemed to be more appropriate than `q`.
For instance, numpy.arange is used `step` parameter for spacing between values. Moreover, torch.arange also adopts this. 

https://docs.scipy.org/doc/numpy/reference/generated/numpy.arange.html
https://pytorch.org/docs/stable/torch.html#torch.arange

Since optuna's users probably are familiar with these, IMO it's better that this parameter is similar to these styles.

Therefore, I would like to modify `q` into `step`. 
If others give feedback to me, it's so helpful for me.